db/migrate/20160209130938_add_service_order_and_user_name_to_miq_request.rb

```diff
@@ -0,0 +1,6 @@
+class AddServiceOrderAndUserNameToMiqRequest < ActiveRecord::Migration
+  def change
+    add_column :miq_requests, :service_order_id, :bigint
+    add_column :miq_requests, :user_name, :string
```

Comment:
Correct, we want to capture the user that created the Service Order so we have some reference if that user object gets deleted.  MiqRequest already has this data which I recall was added to avoid additional lookups when viewing requests but also works for this purpose as well.